src/Qowaiv/Web/HttpMethodParser.cs

```diff
@@ -0,0 +1,41 @@
+using System.Net.Http;
+
+namespace Qowaiv.Web;
+
+/// <summary>Helper method to facility parsing for an <see cref="HttpMethod"/>.</summary>
+/// <remarks>
+/// Required to support .NET 6.0 and earlier.
+/// </remarks>
+internal static class HttpMethodParser
+{
+#if NET8_0_OR_GREATER
+    /// <inheritdoc cref="HttpMethod.Parse(ReadOnlySpan{char})" />
+    [Pure]
+    public static HttpMethod? Parse(string? str)
+        => str is { Length: > 0 }
+        ? HttpMethod.Parse(str)
+        : null;
+#else
+    /// <summary>Parses an <see cref="HttpMethod"/>.</summary>
```

Comment:
```suggestion
    /// <summary>Parses a <see cref="HttpMethod"/>.</summary>
```

Reply:
That's is not correct is it? The 'h' in HTTP is a silent one: aitch, tee, tee, pee. So I would argue 'an'.